plugins/modules/pnp_intent.py

```diff
@@ -888,22 +932,24 @@ class instance for further use.
         ):
             self.result['response'] = self.have.get("device_found")
             self.result['msg'] = "Device is already claimed"
+            self.log(self.result['msg'], "WARNING")
             if update_response.get("deviceInfo"):
                 self.result['changed'] = True
                 return self
 
         claim_params = self.get_claim_params()
-        self.log(str(claim_params))
+        self.log("Paramters for claiming the device are {0}".format(str(claim_params)), "DEBUG")
```

Comment:
self.log("Parameters for claiming the device: {0}".format(str(claim_params)), "DEBUG")

Reply:
Resolved